api/api.go

```diff
@@ -169,6 +169,12 @@ func NewAPIWithVersion(ctx context.Context, globalConfig *conf.GlobalConfigurati
 					r.Get("/", api.adminUserGet)
 					r.Put("/", api.adminUserUpdate)
 					r.Delete("/", api.adminUserDelete)
+					r.Route("/mfa", func(r *router) {
+						r.Put("/update_factor_status", api.adminUserUpdateFactorStatus)
+						r.Delete("/delete_factor", api.adminUserDeleteFactor)
+						r.Put("/disable", api.DisableMFA)
+						r.Put("/enable", api.EnableMFA)
```

Comment:
Same here -- can we make this more RESTful? Example: to disable a factor you send a PUT on `/mfa/<factor>`; or to delete it a DELETE on `/mfa/<factor>`?

Sorry if this is handled in the RFC, but it's kind-of standing out from the rest of the API in GoTrue.